File: 0389-find-the-difference/0389-find-the-difference.cpp
class Solution {
public:
    char findTheDifference(string s, string t) {
        int tSum =0;
        int sSum = 0;
        for(int i=0;i<t.size();i++){
            tSum += t[i];
        }
        for(int i=0;i<s.size();i++){
            sSum += s[i];
        }
        char ans = (char)(tSum - sSum);
        return ans;
    }
};

File: 0389-find-the-difference/0389-find-the-difference.java
class Solution {
    public char findTheDifference(String s, String t) {
        int n = t.length();
        int m = s.length();
        int tSum =0;
        int sSum =0;
        for(int i=0;i<n;i++){
            tSum += (int)t.charAt(i);
        }
        for(int j=0;j<m;j++){
            sSum += (int)s.charAt(j);
        }
        char ans = (char)(tSum-sSum);
        return ans;

    }
}